apps/server/src/modules/learnroom/controller/dto/board/board-skeleton-card.response.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class BoardSkeletonCardReponse {
+	constructor({ cardId, height }: BoardSkeletonCardReponse) {
+		this.cardId = cardId;
+		this.height = height;
+	}
+
+	@ApiProperty({
+		pattern: '[a-f0-9]{24}',
+	})
+	cardId: string;
+
+	@ApiProperty({
+		description:
+			'The approximate height of the referenced card. Intended to be used for prerendering purposes. Note, that different devices can lead to this value not being precise',
+	})
+	height: number;
```

Comment:
why we want to store this in the backend? Each card type could have some pre-defined css for the skeleton to load instead? The card type besides the id could be sufficient to render the skeleton in front-end?